haskell/ghc_bindist_build_template.bzl

```diff
@@ -0,0 +1,15 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
This is not a Skylark source file, so should not have the `.bzl` file extension. These kings of templates are typically called `BUILD.ghc` or somesuch.